### PR TITLE
[inlinehelp] Add inlineHelp Toggle Button control to modules and plugins

### PR DESCRIPTION
**Pull request by conseilgouz**
### Summary of Changes

This PR adds inlineHelp toggle button to modules and plugins options to give users the capability to show/hide parameters descriptions.
This PR improves #35610 and #37849
Instead of always displaying descriptions for modules and plugins, it becomes a user choice. In case of a doubt about a parameter, just click on the button.

Note : it could be improved by adding this to all components. For example, Featured articles menu, Article Class has a 4 lines description, usefull once. 

### Testing Instructions
Apply PR #37849 before applying this PR.

- Check that inlineHelp toogle button is present and working in modules parameters.
- Check that inlineHelp toogle button is present and working in admin modules parameters.
- Check that inlineHelp toogle button is present and working in plugins parameters.
- Check that inlineHelp button is not shown if plugin has no description to display (for example content page break)

### Actual result BEFORE applying this Pull Request
After #37849 , the default behaviour is "show all descriptions" for modules and plugins parameters if no inlineHelp toggle button is displayed. 

### Expected result AFTER applying this Pull Request

InlineHelp toogle button is displayed and working for modules/plugins if parameters have descriptions. It is hidden otherwise.

### Documentation Changes Required
none

### CODE REVIEW

**Comment by richard67** [comment]
@conseilgouz Does it need to apply #37849 , too, for testing this PR? If so, you should mention that in the testing instructions.

**Comment by conseilgouz** [comment]
> @conseilgouz Does it need to apply #37849 , too, for testing this PR? If so, you should mention that in the testing instructions.

Done

File: administrator/components/com_plugins/src/View/Plugin/HtmlView.php [hunk 1]
```diff
@@ -113,5 +113,6 @@ protected function addToolbar()
 		}
 
 		ToolbarHelper::help($help->key, false, $url);
+		ToolbarHelper::inlinehelp();
```
Comment (Quy) [inline administrator/components/com_plugins/src/View/Plugin/HtmlView.php]:
```suggestion
		ToolbarHelper::inlinehelp();
		ToolbarHelper::help($help->key, false, $url);
```

**Comment by Quy** [comment]
Edit `Action Log - Joomla` plugin.
Preferably don't display the `Toggle Inline Help` button as there are no settings to display inline help.

Edit `CAPTCHA - Invisible reCAPTCHA` plugin.
Click `Toggle Inline Help` button.
Nothing happens.

**Comment by conseilgouz** [comment]
> Edit `Action Log - Joomla` plugin. Preferably don't display the `Toggle Inline Help` button as there are no settings to display inline help.

Button is now hidden if no description is available in xml

> Edit `CAPTCHA - Invisible reCAPTCHA` plugin. Click `Toggle Inline Help` button. Nothing happens.

I was not able to reproduce this issue. When I click on button, descriptions are shown/hidden/shown....

**Comment by cyrez** [comment]
@conseilgouz Have tested, but not working for plugins and admin modules, because the <inlinehelp/> tag is not at the right place.
See my comment in files changed.
;-)

**Comment by conseilgouz** [comment]
> @conseilgouz Have tested, but not working for plugins and admin modules, because the tag is not at the right place. See my comment in files changed. ;-)

@cyrezdev : done

**Comment by cyrez** [comment]
> > @conseilgouz Have tested, but not working for plugins and admin modules, because the tag is not at the right place. See my comment in files changed. ;-)
> 
> @cyrezdev : done

Modules admin needed too : https://github.com/joomla/joomla-cms/pull/37915/files#diff-39a20a749da8e579b9b531f02b6648028489795fda3094a78db422b4af5bc44a

If you test, you will see that it does not work with admin modules...

**Comment by cyrez** [comment]
> Apply PR #37849 before applying this PR.

Note: not needed to apply the PR #37849 before tested this one, as it is an improvement that will not change the behavior of your changes here. cc/ @richard67

**Comment by conseilgouz** [comment]
> > > @conseilgouz Have tested, but not working for plugins and admin modules, because the tag is not at the right place. See my comment in files changed. ;-)
> > 
> > 
> > @cyrezdev : done
> 
> Modules admin needed too : https://github.com/joomla/joomla-cms/pull/37915/files#diff-39a20a749da8e579b9b531f02b6648028489795fda3094a78db422b4af5bc44a
> 
> If you test, you will see that it does not work with admin modules...

Yes, I missed this one. Done. Thank You.

**Comment by cyrez** [comment]
I have tested this item :white_check_mark: successfully on 473631f693d87fe4d591985e7e85f2a27ef7ce18<br /><br />Needs to apply script change manually when using com_patchtester.
<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37915">issues.joomla.org/tracker/joomla-cms/37915</a>.</sub>

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 473631f693d87fe4d591985e7e85f2a27ef7ce18<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37915">issues.joomla.org/tracker/joomla-cms/37915</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37915">issues.joomla.org/tracker/joomla-cms/37915</a>.</sub>

**Comment by bembelimen** [comment]
Thx

**Pull request merged**